app/code/community/Bolt/Boltpay/controllers/ApiController.php

```diff
@@ -31,26 +31,7 @@ public function hookAction()
     {
```

Comment:
maybe I don't understand how trait works; don't you have to specify that this class use ApiTrait?

Reply:
Good catch!!!
I made an error here.  Breaking these mPRs down is very demanding and I missed porting that over.

Reply:
`use` statement added.

Reply:
use statement added